docs/wiki/Solution creation.md

```diff
@@ -267,28 +225,40 @@ module rg 'br/modules:microsoft.resources.resourcegroups:0.4.735' = {
 }
 
 // Network Security Group
-module nsg 'br/modules:microsoft.network.networksecuritygroups:0.4.735' = {
+module nsg 'br/modules:microsoft.network.networksecuritygroups:1.0.0' = {
   name: 'registry-nsg'
   scope: resourceGroup(resourceGroupName)
   params: {
-    name: networkSecurityGroupName
+    name: 'defaultNsg'
   }
   dependsOn: [
     rg
   ]
 }
 
 // Virtual Network
-module vnet 'br/modules:microsoft.network.virtualnetworks:0.4.735' = {
+module vnet 'br/modules:microsoft.network.virtualnetworks:1.0.0' = {
   name: 'registry-vnet'
   scope: resourceGroup(resourceGroupName)
   params: {
-    name: vnetName
-    addressPrefixes: vNetAddressPrefixes
-    subnets: subnets
+    name: 'defaultVNET'
+    addressPrefixes: [
+      '10.0.0.0/16'
+    ]
+    subnets: [
+      {
+        name: 'PrimarySubnet'
+        addressPrefix: '10.0.0.0/24'
+        networkSecurityGroupName: nsg.name
+      }
+      {
+        name: 'SecondarySubnet'
+        addressPrefix: '10.0.1.0/24'
+        networkSecurityGroupName: nsg.name
+      }
+    ]
   }
   dependsOn: [
```

Comment:
Isn't this indirectly achieved by subnets depending on nsg which in turns depends on rg?

Reply:
Yes., but it would be transient and hence potentially harder to understand. Thoughts?

Reply:
I'd avoid code which is not necessary, but I don't find that hard to understand so maybe my opinion is biased. Besides that I don't have other thoughts

Reply:
don't have a strong opinion about it - and you at least have an opinion, so I updated it now :D